gnome-shell/src/gnome-shell-sass/_palette.scss

```diff
@@ -39,5 +39,5 @@ $darkblue: #335280;
 // Semantic colors
 $primary_accent_bg_color: $orange;
 $primary_accent_fg_color: $white;
-$secondary_accent_bg_color: $light_aubergine;
+$secondary_accent_bg_color: $orange;
```

Comment:
This `secondary` accent system can also be removed imo.

Reply:
Hmmm, two questions:

1) don't we need those colors somewhere for the yaru flavours?
2) if we remove the secondary accent color can't we remove that additional abstraction of accent colors alltogether and stick to selected bg color like upstream gtk3?

Reply:
I would prefer keep this abstraction in the `_palette` file to keep the symlink system based flavour.
In my mind, I think we should rename both `$primary_accent_...` and `$secondary_accent_...` to just `$accent_...`.